python/setup.py

```diff
@@ -45,7 +45,7 @@ def collect_requirements():
 
 setuptools.setup(
     name="felix",
-    version="2.0.0-beta3",
+    version="2.0.0.dev1",
```

Comment:
Python, OTOH, does special-case some pre-release labels so `2.0.0.dev1 < 2.0.0-beta3`.  One option that will sort after the beta but before the next beta/full release is `2.0.0b4.dev1`: See: https://www.python.org/dev/peps/pep-0440/#developmental-releases